documentation/prerequisites.md

```diff
@@ -35,6 +36,7 @@ Following table summarizes key features introduced in vSphere Cloud Provider in
     - With kuberneters version 1.9.x if user wants to use multiple vCenters then vSAN storage can not be used. vSAN is limited to one cluster in one vCenter deployment.
 * vCenter user with required set of privileges.
 * VMware Tools needs to be installed on the guest operating system on each Node VM. Please refer this [link](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.html.hostclient.doc/GUID-ED3ECA21-5763-4919-8947-A819A17980FB.html) for instruction on installing VMware tools.
+* Virtual Hardware Version (VM version) 11 is required for VMs serving as Container Hosts.
```

Comment:
I suggest we reword as follows - Virtual Hardware Version (VM version) 11 & onwards is required for VMs serving as Container Hosts. 

Once @divyenpatel files a tracker issue for Virtual Hardware Version 13 compatibility I suggest updating the known issues section to reflect that.

Reply:
@tusharnt  issue for `Version 13 compatibility` is tracked at - https://github.com/vmware/kubernetes/issues/450

Reply:
Virtual Hardware Version (VM version) 11 & onwards gives wrong message.
Virtual Hardware Version (VM version) 11 only works and tested.